src/hydro/srcterms/tabular_cooling.hpp

```diff
@@ -116,6 +116,9 @@ class TabularCooling {
   CoolIntegrator integrator_;
 
   // Temperature floor (assumed in Kelvin and only used in cooling function)
+  // This is either the temperature floor used by the hydro method or the
+  // lowest temperature in the cooling table (assuming zero cooling below the
+  // table), whichever temperature is higher
```

Comment:
We might want to discuss if it'd make sense to introduce a distinction between those "floors".
For example, we may want to have expanding plasma that "cools" (so actually can get colder without active cooling) without being reset to the floor.

Reply:
This has been fixed in #28, correct?